### PR TITLE
help pages for provider and advocate 

### PR DESCRIPTION
**Pull request by nobane**

### CODE REVIEW

File: _15thnight/api/account.py [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
 from _15thnight.models import Service, User
-from _15thnight.queue import queue_send_email
-from _15thnight.util import jsonify, api_error
+from _15thnight.util import api_error, jsonify, required_access
 
 try:
     from config import RESET_TOKEN_LIFE
 except:
-    from config import RESET_TOKEN_LIFE
+    from configdist import RESET_TOKEN_LIFE
```
Comment (justinnoah) [inline _15thnight/api/account.py:20]: Ha, nice catch

**Comment by justinnoah** [comment]
**[Question/Discussion]** While a good refactoring idea, since the routes do not specifically pertain to the help pages themselves, those changes should probably be part of another commit/issue

File: _15thnight/core.py [hunk 2]
```diff
@@ -1,10 +1,17 @@
...
+from flask_mail import Message
 
-from _15thnight.queue import queue_send_message
+from _15thnight.queue import queue_send_message, queue_send_email
 from _15thnight.models import (
     Alert, Need, NeedProvided, ProviderNotified, Service, Response, User
 )
 
+try:
+    from config import SUPPORT_EMAIL
```
Comment (justinnoah) [inline _15thnight/core.py]: If possible, it'd be better if the SUPPORT_EMAIL value were grabbed from the app's config.

```python
from flask import current_app

support_email = current_app.config.get("SUPPORT_EMAIL")

File: _15thnight/core.py [hunk 3]
```diff
@@ -130,3 +137,56 @@ def send_out_resolution(need):
...
+    Send a help message.
+    """
+    sender = (user.name or '', user.email)
+    message = Message(
+        sender=sender,
+        reply_to=sender,
+        subject="15th Night RAN Website Support Request",
+        body=render_template('email/support_request.txt',
+            provider=user, message=message),
+        recipients=[SUPPORT_EMAIL]
```
Comment (justinnoah) [inline _15thnight/core.py]:
```python
recipients=[current_app.config.get("SUPPORT_EMAIL","")]

File: client/src/components/chrome/index.css [hunk 4]
```diff
@@ -5,6 +5,6 @@
         }
     }
     .container-fluid {
-        padding-top: 83px;
+        padding-top: 93px;
```
Comment (justinnoah) [inline client/src/components/chrome/index.css:8]: **facepalm** (no actual issue, just magic numbers changes are...lame, but that's CSS for you)

**Comment by nobane** [comment]
@justinnoah in regard to the URL refactor being elsewhere, I agree, but for now could we just merge it in? It's a fairly simple refactor.

**Comment by justinnoah** [comment]
After one-on-one discussion with @spectralsun about the changes, this going to be merged as is because we are trying to move exceptionally quick right now. However, future PRs will have to be changed.

**Pull request merged**